src/states/node.rs

```diff
@@ -1995,8 +1995,16 @@ impl Node {
 
     /// Informs our peers that our section's member list changed. If `dst_prefix` is `Some`, only
     /// tells that section, otherwise tells all connected sections, including our own.
```

Comment:
Well there was a comment added to `RoutingTable::check_invariant()` which explains it OK I guess?  Or do you think we should still add a bit more info here too?